src/components/structures/TimelinePanel.js

```diff
@@ -108,7 +108,7 @@ var TimelinePanel = React.createClass({
 
     getDefaultProps: function() {
         return {
-            timelineCap: 250,
+            timelineCap: 1000,
```

Comment:
WFM! Although we might want to update the test that was failing before so that it runs with a more realistic `timelineCap` set. (At the moment it sets the cap to 100).